### PR TITLE
Throwing tweaks / indirect floor tile+rod buff

### PR DESCRIPTION
**Pull request by MrPerson**
Throwing a stack only throws one item in that stack instead of the whole stack. This will obviously make a stack of floor tiles very deadly because you can just keep throwing them very quickly. This mechanic does not apply to the following stacks: Telecrystals, medical stuff, wrapping paper, and cable coil.

Throwing paper automatically turns it into a paper plane. Picking up a paper plane turns it back into paper.

:cl:
add: Throwing a stack will throw one item in the stack instead of the whole stack. The rest of that stack of floor tiles will remain in your hand.
/:cl:

### CODE REVIEW

**Comment by Mindustry** [comment]
> Throwing paper automatically turns it into a paper plane. Picking up a paper plane turns it back into paper.

I don't like this because it's not very realistic compared to what we have now but the rest is neato

**Comment by lzimann** [comment]
this seems a bit too strong imo

**Comment by Kevak** [comment]
Can you maybe make it not apply to material stacks (Metal, plasma, glass, uranium, diamond, and so on) as well? Tossing those is regularly used by science to move stuff around.

**Comment by RandomMarine** [comment]
There's got to be a more elegant solution than arbitrarily deciding which stacks throw individually or together. Something that puts that control in the hands of the player.

**Comment by Iamgoofball** [comment]
if i throw paper it should paper plane, don't listen to these bozos

**Comment by Grazyn** [comment]
This is just a buff to floor tiles and a hindrance for everything else. Expecially for science, engineers and anything construction-related. The idea itself is kinda neat but it should be preference you can toggle in game prefs, so I can turn it on when it's bloodcult or construct wizard

**Comment by FantasticFwoosh** [comment]
I dont like this at all, if you just want to throw 1 tile or anything between 50 tiles you can just alt click, quickly type the number and then throw it, this penalises people passing over materials quickly and over-complicated.

> Throwing is a common way before the damage PR made things take damage, of putting things onto surfaces without having to do so manually by clicking on sometimes cluttered tables

OR (if no hotkey enabled)

Using your opposite free hand click on the stack to draw 1 automatically

& 

You can just click alt to turn any paper into a airoplane, why does it need to be transformed mid flight?

> TLDR "I too unrobust & ded, pls buff"

**Comment by Haevacht** [comment]
If it worked like beakers with the transfer amounts, that'd be great.

Auto planes makes my dream of coating anywhere near a paper bin in planes a reality.

**Comment by jughu** [comment]
I mean throwing papers should auto plane yes it makes sense you cant throw flat paper so you turn into a paper plane so it can fly because if you dont it falls slowy down on the ground and then you have to pick it up again and make it a paper plane by alt clicking and that be just stupid right?

**Comment by GunHog** [comment]
:-1: because throwing stacks of materials would be substantially limited by this.

**Comment by jughu** [comment]
👎to stack throwing
👍 auto paper planes

**Comment by KorPhaeron** [comment]
Conflicting and this would actually be horrendous balance wise, someone should PR paper planes on their own though
